src/factoid.rs

```diff
@@ -118,7 +150,7 @@ mod tests {
         assert_eq!(brain.respond("foo"), Some("bar".to_string()));
 
         // Set arbitrarily to make the test work
-        brain.set_rng_seed(69);
+        brain.set_rng_seed(696969);
```

Comment:
random choice through an iterator works a bit differently than selecting from a vec, so i had to find a new random seed

Reply:
this seems much nicer!